### PR TITLE
Add homepage attribute as mandatory & topics as recommended

### LINKED ISSUE #88: [conan center] check homepage attribute is specified

### PR DESCRIPTION
**Pull request by uilianries**
- Add `homepage` attribute as mandatory & `topics` as recommended

closes #88
Wiki:

#### **<a name="KB-H003">#KB-H003</a>: "RECIPE METADATA"**

The recipe has to declare the following fields: [url](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#url), [license](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#license) , [description](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#description), [homepage](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#homepage). Also, we recommend adding [topics](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#topics)

### CODE REVIEW

**Comment by uilianries** [comment]
@memsharded  what about moving `author` and `topics` to warning level?

**Comment by memsharded** [comment]
> @memsharded what about moving author and topics to warning level?

No, author should be an error, better not having authors there. Attribution is done via Github mechanisms, not an email of one contributor in a recipe.

Topics can be a warning, yes. My problem with topics is that this does not longer seem used by engines like elasticsearch, which works fine with the whole text descriptions.

**Comment by danimtb** [comment]
Please @uilianries stick to the issue this PRs tries to close (#88) that way we don't duplicate efforts with other ones (#97) and it is easier to review and implement

**Comment by uilianries** [comment]
@danimtb good idea! I gonna update this PR!

File: tests/test_hooks/conan-center/test_conan-center.py [hunk 1]
```diff
@@ -16,6 +16,9 @@ class AConan(ConanFile):
             url = "fake_url.com"
             license = "fake_license"
             description = "whatever"
+            author = "unknown"
```
Comment (danimtb) [inline tests/test_hooks/conan-center/test_conan-center.py]: Is this needed for this PR?

**Comment by danimtb** [comment]
I think this PR is not closing #97 (it shouldn't), so please remove it from the description

File: tests/test_hooks/conan-center/test_conan-center.py (hunk 1 above)
Comment (uilianries) [reply tests/test_hooks/conan-center/test_conan-center.py]: oh it's from first, since my initial idea was include author as mandatory attribute.

**Comment by uilianries** [comment]
> I think this PR is not closing #97 (it shouldn't), so please remove it from the description

@danimtb done

File: tests/test_hooks/conan-center/test_conan-center.py (hunk 1 above)
Comment (uilianries) [reply tests/test_hooks/conan-center/test_conan-center.py]: done, I've removed author and topics.

**Comment by uilianries** [comment]
Conflicts were solved!

**Comment by uilianries** [comment]
> > @memsharded what about moving author and topics to warning level?
> 
> No, author should be an error, better not having authors there. Attribution is done via Github mechanisms, not an email of one contributor in a recipe.
> 
> Topics can be a warning, yes. My problem with topics is that this does not longer seem used by engines like elasticsearch, which works fine with the whole text descriptions.

@memsharded done!

**Comment by danimtb** [comment]
Merging before CI end as changes are just related to syntax

**Comment by danimtb** [comment]
I have updated the wiki here https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#kb-h003-recipe-metadata

**Pull request merged**